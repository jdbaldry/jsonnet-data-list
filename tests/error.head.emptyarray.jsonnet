local l = import '../main.libsonnet';

l.head([])
