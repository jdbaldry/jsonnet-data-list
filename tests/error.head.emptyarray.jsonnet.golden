RUNTIME ERROR: empty array: `head`
	../main.libsonnet:(12:5)-(13:9)	function <anonymous>
	error.head.emptyarray.jsonnet:3:1-11	
